internal/pgkeyvalue/pgkeyvalue.go

```diff
@@ -164,6 +256,48 @@ func (c *PGKeyValueStore) Get(ctx context.Context, key string) ([]byte, error) {
 	return *value, nil
 }
 
+func (c *PGKeyValueStore) GetBatch(ctx context.Context, keys []string) (map[string][]byte, error) {
```

Comment:
Is this actually used? 

It isn't obvious if it, but I am only looking via Github so maybe its just non obvious and it actually implements some interface

Reply:
No it's not- this was an api call I added while testing an alternative approach and forgot to remove.  Done now.